cf/read_write/netcdf/netcdfread.py

```diff
@@ -181,6 +182,12 @@ def _create_data(
 
                 .. versionadded:: 3.15.0
 
+           compression_index: `str`, optional
```

Comment:
Sorry I missed this on the corresponding cfdm PR (where we have merged the same description), but a Boolean seems to be expected?

```suggestion
           compression_index: `bool`, optional
```